scripts/output/single/reporting.R

```diff
@@ -77,12 +77,31 @@ message("start generation of EDGE-T reporting")
                                   scenario_title = scenario, model_name = "REMIND",
                                   gdx = paste0(outputdir,"/fulldata.gdx"))
 
-  write.mif(EDGET_output, remind_reporting_file, append=T)
-  deletePlus(remind_reporting_file, writemif=T)
+  write.mif(EDGET_output, remind_reporting_file, append = TRUE)
+  deletePlus(remind_reporting_file, writemif = TRUE)
 
 message("end generation of EDGE-T reporting")
 }
 
+configfile <- file.path(outputdir, "config.Rdata")
```

Comment:
I realize only now that this has the disadvantage of making reporting.R dependent on the existence of the config file. There is, however, no other way, unless we write the path to the magpie report into the gdx, which is a bit too much for now, I think. So, long story short: I'm fine with this solution :-)

Reply:
`reporting.R` depends on `config.Rdata` anyhow: It calls [`lucode2::getScenNames()`](https://github.com/pik-piam/lucode2/blob/master/R/getScenNames.R) which reads from it.

Reply:
Ah, missed that. Thanks!